src/test/java/shop/RealItemTest.java

```diff
@@ -0,0 +1,28 @@
+package shop;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class RealItemTest {
+    private RealItem realItem;
+
+    @BeforeEach
+    public void setUp() {
+        realItem = new RealItem();
+    }
+
+    @Test
+    void getWeight() {
+        final double expected = 20.99;
+        realItem.setWeight(20.99);
+        assertEquals(expected, realItem.getWeight());
```

Comment:
It was a trap. It doesn't make sense to create tests for getters/setters/etc. For RealItem/VirtualItem classes there are 2 correct solutions here:
1. Do not implement unit tests at all for them
2. Add test for toString method as far as it was overriden

Please update